packages/amplify_authenticator/example/lib/main.dart

```diff
@@ -34,6 +34,15 @@ class _MyAppState extends State<MyApp> {
   void _configureAmplify() async {
     try {
       await Amplify.addPlugin(AmplifyAuthCognito());
+      // Uncomment this block, and comment out the one above, in order to persist credentials
```

Comment:
I would use a different example since indexedDB will become the default option: https://github.com/aws-amplify/amplify-flutter/pull/1749

Reply:
I feel like this should be in the auth example, too, not the authenticator example.